src/test/java/uk/gov/pay/ledger/transaction/service/TransactionServiceTest.java

```diff
@@ -150,29 +150,29 @@ public void shouldListTransactionsWithCorrectQueryParamsForPaginationLinks() {
         when(mockTransactionDao.searchTransactions(any(TransactionSearchParams.class))).thenReturn(transactionViewList);
         when(mockTransactionDao.getTotalForSearch(any(TransactionSearchParams.class))).thenReturn(10L);
 
-        searchParams.setEmail("test@email.com");
-        searchParams.setCardHolderName("test");
-        searchParams.setFromDate("2019-05-01T10:15:30Z");
-        searchParams.setToDate("2019-06-01T10:15:30Z");
-        searchParams.setReference("ref");
-        searchParams.setFirstDigitsCardNumber("4242");
-        searchParams.setLastDigitsCardNumber("1234");
-        searchParams.setPaymentStates(new CommaDelimitedSetParameter("created,submitted"));
-        searchParams.setRefundStates(new CommaDelimitedSetParameter("created,refunded"));
-        searchParams.setCardBrands(new CommaDelimitedSetParameter("visa,mastercard"));
+        setAllSearchParams();
 
         TransactionSearchResponse transactionSearchResponse = transactionService.searchTransactions(searchParams, mockUriInfo);
-        PaginationBuilder paginationBuilder = transactionSearchResponse.getPaginationBuilder();
-        String selfLink = paginationBuilder.getSelfLink().getHref();
+        assertCorrectPaginationQueryParams(transactionSearchResponse);
+    }
```

Comment:
To be consistent with the other test it would be good to verify the "old" methods in this test.
```
verify(mockTransactionDao).searchTransactions(searchParams);
verify(mockTransactionDao).getTotalForSearch(searchParams);
```

Reply:
Will address in next PR